src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -110,26 +110,28 @@ const subsetResult = (
     }, {})
 
   const tables: SubsetTable[] = []
-  let lastTable
+  let lastTable = ''
 
   // group by table id (series)
   for (let ni = 0; ni < size; ni++) {
     if (
-      `y${subset['result'][0].data[ni]}:t${subset['table'][0].data[ni]}` ===
+      `y${subset['result']?.[0]?.data?.[ni]}:t${subset['table']?.[0]?.data?.[ni]}` ===
```

Comment:
`lastTable` is set to an empty string to make it clear that it's different than `undefined`.

The interpolated string `y${subset['result']?.[0]?.data?.[ni]}:t${subset['table']?.[0]?.data?.[ni]}` may be `yundefined:tundefined`. This looks bizarre and could be incorrectly interpreted in the future as intended to be `undefined`. This is the reason why we set an initial empty string value for `lastTable` above.